tests/src/Interop/ArrayMarshalling/SafeArray/RecordNative.cpp

```diff
@@ -0,0 +1,399 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <xplatform.h>
+#include <oleauto.h>
+#include <algorithm>
+#include "platformdefines.h"
```

Comment:
Is `platformdefines.h` in this directory? I think this should be `<...>`? Please fix this where ever you see it.